test/software/amazon/ion/EquivsTestCase.java

```diff
@@ -164,21 +172,17 @@ protected void runEquivalenceChecks(final IonDatagram datagram,
      */
     protected void checkEquivalence(final IonValue left,
```

Comment:
None of the changes in this method actually add anything, they just pollute the diff. If we're going to make style changes to existing code (which isn't a priority), it should be separated from commits that make behavioral changes.